src/v/kafka/server/usage_aggregator.h

```diff
@@ -32,7 +32,7 @@ std::chrono::time_point<clock_type, duration> round_to_interval(
     /// error if this cannot be done within some threshold.
     using namespace std::chrono_literals;
     const auto interval = usage_window_width_interval;
-    const auto err_threshold = interval < 2min ? interval : 2min;
+    const auto err_threshold = interval < 2min ? 1s : 2min;
```

Comment:
@dotnwat I would also consider we lower this from 2mins to some much smaller value. If there is 60 second drift we'd essentially be rounding the timestamp and covering up a potential issue.

Reply:
i'm a bit confused about what is happening. if we are choosing a tighter bound to deal with drift, won't the problem be more likely to happen?

Reply:
I would keep it to reduce number of dropped windows. The best would be to have it configurable so we can change it if there's any problem with that but I understand it's much more work 😅

Reply:
I don't think the problem is more likely to occur, since in the testing after the fix, reported drift is always 0 seconds.